apps/arena/lib/arena/game_updater.ex

```diff
@@ -58,45 +58,31 @@ defmodule Arena.GameUpdater do
   def handle_info(:update_game, %{game_state: game_state} = state) do
     Process.send_after(self(), :update_game, state.game_config.game.tick_rate_ms)
 
-    entities_to_collide = Map.merge(game_state.players, game_state.projectiles)
+    # only use alive players
```

Comment:
We could turn this comment into an issue and reference it with `# #246` instead. This way we can keep the detailed context of this problem.